### PR TITLE
REG-2369: Initial test support for PAYE service

### PR DESCRIPTION
**Pull request by nigelhp**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 ONS Digital
```
Comment (awharris) [inline LICENSE]:
```suggestion
Crown Copyright (c) 2018 Office for National Statistics
```

File: src/main/scala/uk/gov/ons/br/test/hbase/HBaseJsonRequestBuilder.scala [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+trait HBaseJsonRequestBuilder extends BasicAuthentication {
+  def getHBaseJson(namespace: String, tableName: String, rowKey: String,
+                   auth: Authorization = Authorization.DummyAuthorization): MappingBuilder =
+    get(urlEqualTo(getUrlFor(namespace, tableName, rowKey))).
+      withHeader(ACCEPT, equalTo(JSON)).
+      withHeader(Authorization.HeaderName, equalTo(Authorization.value(auth)))
+
+  private def getUrlFor(namespace: String, tableName: String, rowKey: String): String =
+    s"/$namespace:$tableName/$rowKey"
```
Comment (awharris) [inline src/main/scala/uk/gov/ons/br/test/hbase/HBaseJsonRequestBuilder.scala:17]: We are assuming there always will be a namespace? I know there will always be a namespace in our deployed envs but could the scenario arise where we need to support a HBase schema, perhaps local where a namespace has not been used or do we just enforce the use of a namespace everywhere?

**Review by awharris — COMMENTED** [review]
Not sure if anything to worry about

[warn] Found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[warn] 	* com.typesafe.akka:akka-stream_2.12:2.5.17 is selected over {2.5.9, 2.4.20}
[warn] 	    +- com.typesafe.play:play-streams_2.12:2.6.20 ()      (depends on 2.5.17)
[warn] 	    +- com.typesafe.akka:akka-http-core_2.12:10.0.14 ()   (depends on 2.4.20)
[warn] 	    +- com.typesafe.play:play-ws-standalone_2.12:1.1.10   (depends on 2.5.9)
[warn] 	* com.typesafe.akka:akka-actor_2.12:2.5.17 is selected over 2.4.20
[warn] 	    +- com.typesafe.akka:akka-slf4j_2.12:2.5.17 ()        (depends on 2.5.17)
[warn] 	    +- com.typesafe.akka:akka-stream_2.12:2.5.17 ()       (depends on 2.5.17)
[warn] 	    +- com.typesafe.play:play_2.12:2.6.20 ()              (depends on 2.5.17)
[warn] 	    +- com.typesafe.akka:akka-parsing_2.12:10.0.14 ()     (depends on 2.4.20)
[warn] 	* junit:junit:4.12 is selected over {3.8.1, 4.11}
[warn] 	    +- com.github.tomakehurst:wiremock:2.19.0             (depends on 4.12)
[warn] 	    +- com.typesafe.play:play-test_2.12:2.6.20 ()         (depends on 4.12)
[warn] 	    +- org.seleniumhq.selenium:selenium-support:3.5.3     (depends on 4.12)
[warn] 	    +- org.seleniumhq.selenium:selenium-java:3.5.3        (depends on 4.12)
[warn] 	    +- com.novocode:junit-interface:0.11                  (depends on 4.11)
[warn] 	    +- org.xmlunit:xmlunit-legacy:2.5.1                   (depends on 3.8.1)

File: LICENSE (hunk 1 above)
Comment (nigelhp) [reply LICENSE]: Updated as suggested, along with update to the year.

File: src/main/scala/uk/gov/ons/br/test/hbase/HBaseJsonRequestBuilder.scala (hunk 2 above)
Comment (nigelhp) [reply src/main/scala/uk/gov/ons/br/test/hbase/HBaseJsonRequestBuilder.scala:17]: As discussed, the assumption has been made that there will always be a namespace.

**Review by awharris — APPROVED** [review]

**Pull request merged**